cmd/containerd-shim/main_unix.go

```diff
@@ -41,6 +41,7 @@ import (
 	shimapi "github.com/containerd/containerd/runtime/v1/shim/v1"
 	"github.com/containerd/ttrpc"
 	"github.com/containerd/typeurl"
+	"github.com/docker/docker/pkg/reexec"
```

Comment:
Using this as a dependency is a big no no. Either reimplement or find another solution.

DRY is a liability here.

Reply:
Yes @stevvooe. I will find other way to handle this. 😄

Reply:
Was looking around for a suggestion, but I think @dmcgowan covered the options in https://github.com/containerd/containerd/issues/2497#issuecomment-407854291.

Reply:
Found a way to create a `Fmountat` which we could use here https://github.com/dmcgowan/containerd/commit/0d7019136bbe7c0708dfca428ba6ffdbef6d9b0e

Reply:
@stevvooe yes. the option provided by @dmcgowan works! I have changed the code. Please take a look.